sdk/ai/azure-ai-client/samples/agents/sample_agents_basics.py

```diff
@@ -29,32 +32,24 @@
     logging_enable=True, # Optional. Remove this line if you don't want to show how to enable logging
 )
 """
-agent = ai_client.agents.create_agent(
-    model="gpt-4-1106-preview", name="my-assistant", instructions="You are helpful assistant"
-)
-print("Created agent, agent ID", agent.id)
-
-thread = ai_client.agents.create_thread()
-print("Created thread, thread ID", thread.id)
-
-message = ai_client.agents.create_message(thread_id=thread.id, role="user", content="Hello, tell me a joke")
-print("Created message, message ID", message.id)
 
-run = ai_client.agents.create_run(thread_id=thread.id, assistant_id=agent.id)
-print("Created run, run ID", run.id)
+with ai_client:
+    agent = ai_client.agents.create_agent(
+        model="gpt-4-1106-preview", name="my-assistant", instructions="You are helpful assistant"
+    )
+    logging.info(f"Created agent, agent ID: {agent.id}")
 
-# poll the run as long as run status is queued or in progress
-while run.status in ["queued", "in_progress", "requires_action"]:
```

Comment:
The while loop in basic sample is intentionally shown, the sample purpose is to show what is the lowest level api operation without convenience methods and which follows closest to openai api

Reply:
Done